docs/getting-started/server/sso/index.md

```diff
@@ -11,6 +11,14 @@ You can set up SSO for development using:
 
 - [a local IdP](./local.md) (recommended)
 - [Okta](./okta.mdx)
+- a local server running the SSO project, e.g.
```

Comment:
It looks like this is handled [here](https://github.com/bitwarden/contributing-docs/blame/5b841fc8c7fdc98de6a19a80aa6677d3a3354613/docs/getting-started/server/sso/local.md) so this block can be removed.